### ISSUE TITLE
set adjust-open width is zooming too much

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#223]
*On GitLab by @bouanto on Sep 3, 2020, 01:29*

---

Hi.
Starting recently, using `set adjust-open width` in the config file will automatically zoom too much PDF files that are opened, which will also freeze the UI.
Here's my version of zathura:
```
zathura 0.4.6
girara 0.3.5 (runtime: 0.3.5)
(plugin) pdf-poppler (0.3.0) (/usr/lib/zathura/libpdf-poppler.so)
```
I use i3wm in ArchLinux.
Thanks to fix this issue.

**Comment by sebastinas** [comment]
This should be fixed in 0.4.7.

**Comment by sebastinas** [comment]
closed
